rust/worker/src/memberlist/memberlist_provider.rs

```diff
@@ -118,6 +117,7 @@ impl Configurable<MemberlistProviderConfig> for CustomResourceMemberlistProvider
 }
 
 impl CustomResourceMemberlistProvider {
+    #[cfg(test)]
```

Comment:
I don't think this is meant to be test-only exposed, its library code that should be available to use